### DISCUSSION TITLE
How to connect to services that uses private endpoint

### DISCUSSION THREAD

**Discussion by baatch** [Azure/data-landing-zone#194] [Q&A] [answered]
I cannot find any good documentation on how one should access the different analytical services that are connected through private endpoint?

Networking is not my strong side, some pointers to how I and organizations should approach this would be welcome.

Thanks

**Comment by marvinbuss** [comment, marked as answer]
Hi @baatch,
Please take a look here: [Connecting to Environments Privately](https://github.com/Azure/data-management-zone/blob/main/docs/guidance/EnterpriseScaleAnalytics-ConnectToEnvironmentsPrivately.md)
This should answer your question and simplifies the setup of the Bastion setup, if you decide to move forward with that.

**Reply by marvinbuss** [reply]
Hi @baatch. Was the link helpful or do you need additional help? Please let me know!

**Reply by baatch** [reply]
@marvinbuss thanks exactly what I was looking for !
